dpnp/dpnp_iface_manipulation.py

```diff
@@ -2093,7 +2097,7 @@ def reshape(a, /, newshape, order="C", copy=None):
     -------
     out : dpnp.ndarray
         This will be a new view object if possible; otherwise, it will
-        be a copy.  Note there is no guarantee of the *memory layout* (C- or
+        be a copy. Note there is no guarantee of the *memory layout* (C- or
         Fortran- contiguous) of the returned array.
 
     Limitations
```

Comment:
I guess we can remove the limitations, because we state supported range of `order` argument in the description.